src/js/structs/UniversePackage.js

```diff
@@ -84,7 +84,9 @@ class UniversePackage extends Item {
   }
 
   getVersion() {
-    return this.get("version");
+    // backport because some actions are removing
+    // version and replacing the package obj with currentVersion
+    return this.get("version") || this.get("currentVersion");
```

Comment:
Maybe it's better to change the places that use `.currentVersion` instead of having this inconsistency.

Reply:
that would be a refactoring and I would need to test other parts besides only the feat implemented here.. should be added on another Jira/tasks so for that reason I included both as of now.